packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/dataservice/ProjectController.java

```diff
@@ -182,13 +227,15 @@ public ResponseEntity<List<Project>> getProjects(
 					}
 				}
 			} catch (final Exception e) {
-				log.error(
-						"Cannot get Datasets, Models, and Publications assets from data-service for project_id {}",
-						project.getId(),
-						e);
+				log.error("Failed to get project author name from postgres db for project {}", project.getId(), e);
 			}
 		});
 
+		if (projects.isEmpty()) {
```

Comment:
If a user has no permissions to visible projects then this projects collection will return an empty list on line 104, skip the iteration, then end up here empty. In this case that isn't a INTERNAL_SERVER_ERROR but instead maybe a 204 NO CONTENT or maybe a 404? That may want to be checked around line 104 then this can stay for the error case here.

Reply:
fixed